components/dropdown/dropdown-content-mixin.js

```diff
@@ -1131,16 +1207,34 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 		if (this.trapFocus) {
 			dropdownContentSlots = html`
-			<d2l-focus-trap
-			@d2l-focus-trap-enter="${this._handleFocusTrapEnter}"
-			?trap="${this.opened}">
-			${dropdownContentSlots}
+			<d2l-focus-trap @d2l-focus-trap-enter="${this._handleFocusTrapEnter}" ?trap="${this.opened}">
+				${dropdownContentSlots}
 			</d2l-focus-trap>`;
 		}
 
+		const positionStyle = {};
+		if (this._position) {
+			for (const prop in this._position) {
+				positionStyle[prop] = `${this._position[prop]}px`;
+			}
+		}
+
 		const dropdown = html`
 			<div class="d2l-dropdown-content-position" style=${styleMap(positionStyle)}>
-					 ${dropdownContentSlots}
+				${dropdownContentSlots}
+			</div>
+		`;
+
+		const pointerPositionStyle = {};
+		if (this._pointerPosition) {
+			for (const prop in this._pointerPosition) {
+				pointerPositionStyle[prop] = `${this._pointerPosition[prop]}px`;
+			}
+		}
+
+		const pointer = html`
```

Comment:
The pointer was moved into this mixin from the subclass `render` methods since the pointer position needs to be calculated and rendered in the same way.